### ISSUE TITLE
No module named 'torchxrayvision.baseline_models.jfhealthcare.model'

### ISSUE THREAD

**Issue by bhatuzdaname** [mlmed/torchxrayvision#43]
I did 
```pip install torchxrayvision```
and in an ipython session, when I do:
```import torchxrayvision as xrv```
I get the following error:
```
In [1]: import torchxrayvision as xrv
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-04203dc2a6b0> in <module>
----> 1 import torchxrayvision as xrv

~/workspace/torchxrayvision/lib/python3.6/site-packages/torchxrayvision/__init__.py in <module>
      1 from . import datasets
      2 from . import models
----> 3 from . import baseline_models
      4 from . import autoencoders

~/workspace/torchxrayvision/lib/python3.6/site-packages/torchxrayvision/baseline_models/__init__.py in <module>
----> 1 from . import jfhealthcare
      2

~/workspace/torchxrayvision/lib/python3.6/site-packages/torchxrayvision/baseline_models/jfhealthcare/__init__.py in <module>
      5 import csv
      6 import numpy as np
----> 7 from .model import classifier
      8 import json
      9 import argparse

ModuleNotFoundError: No module named 'torchxrayvision.baseline_models.jfhealthcare.model'
```

**Comment by ieee8023** [comment]
What version are you using? Try pip install torchxrayvision --upgrade

**Comment by ieee8023** [comment]
Sorry this was a bug with 0.0.23 and is now fixed in 0.0.24!

**Comment by bhatuzdaname** [comment]
Thank you!
